src/js/media-time-display.js

```diff
@@ -0,0 +1,36 @@
+import MediaTextDisplay from './media-text-display.js';
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import { formatTime } from './utils/time.js';
+import { Document as document } from './utils/server-safe-globals.js';
+// Todo: Use data locals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString
+
+class MediaTimeDisplay extends MediaTextDisplay {
+  constructor() {
+    super();
+    this.mediaCurrentTimeSet(0);
```

Comment:
Since `mediaCurrentTimeSet` calls `_update` which calls `innerHTML`, I think we will need to move this into the `connectedCallback` for the [reasons specified here](https://github.com/muxinc/media-chrome/pull/38#discussion_r667224184)